apps/frontend/src/app/pages/profile/profile.component.html

```diff
@@ -10,6 +10,15 @@
             @if (!isLocal) {
               <m-report-button [reportType]="ReportType.USER_PROFILE_REPORT" [reportData]="user.id.toString()" />
             }
+
+            @if (canSeeAdminActivity()) {
+              <div>
```

Comment:
Pointless div?

Reply:
Yes. Didn't notice that when I was copying Edit Profile button. Should I remove <div> there too?

Reply:
Yeah go for it